Library/Formula/caf.rb

```diff
@@ -0,0 +1,52 @@
+class Caf < Formula
+  homepage "http://actor-framework.org"
+  head "https://github.com/actor-framework/actor-framework.git"
+
+  stable do
+    url "https://github.com/actor-framework/actor-framework/archive/0.12.0.tar.gz"
+    sha256 "294fb00508c166595d479e9af2fae28f2d370ed164cd1de079ae247acc916801"
+  end
+
+  devel do
+    url "https://github.com/actor-framework/actor-framework.git",
+      :branch => "develop"
```

Comment:
What I wanted to implement is that `--HEAD` uses the master branch and `--devel` the development branch. Is there a better way to do this? Using `devel do` feels a little bit like abusing functionality, since it's not technically a development *release*.